forui/lib/src/theme/theme_data.dart

```diff
@@ -46,6 +49,7 @@ class FThemeData with Diagnosticable {
     required this.boxStyle,
     required this.separatorStyles,
     required this.switchStyle,
+    required this.textFieldStyle,
```

Comment:
We should order these based on how the folders are ordered. Eg. in this case, `textFieldStyle` should come after `headerStyle`